wpstarter/templates/.env.example

```diff
@@ -227,12 +227,10 @@ WP_HOME=
 #WP_PROXY_USERNAME
 #WP_PROXY_PASSWORD
 #WP_PROXY_BYPASS_HOSTS
-#WP_ACCESSIBLE_HOSTS
-#WP_HTTP_BLOCK_EXTERNAL
 
 
 #--------------------------------------------------------------------------------------------------#
 #                                          MISCELLANEOUS                                           #
 #--------------------------------------------------------------------------------------------------#
 #WP_MAIL_INTERVAL=300
-#WP_DEFAULT_THEME=twentyfifteen
\ No newline at end of file
+#WP_DEFAULT_THEME=twentyfifteen
```

Comment:
You mean tagging a new version? Ping me tomorrow, then I'll add a new patch level version and fix the wrong v2.1.1 as well. Thanks in advance!